README.md

```diff
@@ -40,34 +40,61 @@ Install the migrations:
 $ rake alchemy_spree:install:migrations
 ```
 
-Migrate the database:
+Do the next step before running the migrations.
 
-```sh
-$ rake db:migrate
-```
 
 ### Authentication system installation
 
-Since Alchemy 3.0 has dropped the authentication from its core, you will need to choose one authentication system. The easiest choice is to use the [alchemy-devise gem](https://github.com/magiclabs/alchemy-devise).
+Since Alchemy 3.0 has dropped the authentication from its core, you will need to choose an  authentication system yourself. There are 2 available options:
 
-To install alchemy-devise:
+1. Use Spree's [Spree Auth Devise](https://github.com/spree/spree_auth_devise)
 
-```ruby
-# Gemfile
-gem 'alchemy-devise', '~> 2.0'
-```
+  If you already have an existing Spree installations, this is your best bet. `gem 'spree_auth_devise'` should already be in your Gemfile.
 
-And then execute:
 
-```sh
-$ bundle
-```
+  To use Spree Auth Devise, instruct Alchemy to use the Spree Auth Devise user class:
 
-Finally, you'll need to instruct Spree accordingly:
+  ```ruby
+  # config/initializers/alchemy.rb
+  Alchemy.user_class_name = 'Spree::User'
+  Alchemy.current_user_method = :spree_current_user
 
-```ruby
-# config/initializers/spree.rb
-Spree.user_class = "Alchemy::User"
+  # Include the Spree controller helpers to render the
+  # alchemy pages within the default Spree layout
+  Alchemy::BaseHelper.send :include, Spree::BaseHelper
+  Alchemy::BaseController.send :include, Spree::Core::ControllerHelpers::Common
+  Alchemy::BaseController.send :include, Spree::Core::ControllerHelpers::Store
+  ```
+
+2. Install [alchemy-devise gem](https://github.com/magiclabs/alchemy-devise)
+
+  Go with this option if you don't have an authentication system in place and you want to add one.
```

Comment:
Better:

"Go with this option if you have an existing Alchemy installation or if you don't have one and don't want to role an authentication system on your own."